training_review/serializers.py

```diff
@@ -15,3 +15,8 @@ class Meta:
 
     def create(self, validated_data):
         return ReviewNoteService.create_review_note(validated_data)
+
+    def validate(self, data):
+        if self.context['request'].user.has_perm('change_record'):
```

Comment:
Validation should be performed against a specific record, not model-level check.
```python
user.has_perm('training_record.change_record', record)
```